jwplayer-appletv-web-app/js/playlist/PlaylistLoader.js

```diff
@@ -89,7 +89,8 @@ var PlaylistLoader = function() {
         mediaItem.mediaid = playlistItem.mediaid;
         mediaItem.description = playlistItem.description;
         mediaItem.title = playlistItem.title;
-        mediaItem.artworkImageURL = _checkScheme(playlistItem.image);
+        mediaItem.artworkImageURL = playlistItem.image ?
+          _checkScheme(playlistItem.image) : "";
```

Comment:
A type error might be thrown when loading the templates if either the title or description is `undefined`, but other than that it shouldn't be causing any major issues right now.